### PR TITLE
update test case bios

### PR DESCRIPTION
**Pull request by brucepatricklee**
remove the machine varient from test case cfg file

Signed-off-by: Jin Li <jil@redhat.com>

### CODE REVIEW

File: libvirt/tests/cfg/bios/boot_integration.cfg [hunk 1]
```diff
@@ -30,9 +31,3 @@
             bios_reboot_timeout = "1000"
             disk_target_dev = 'hda'
             disk_target_bus = 'ide'
-    variants:
-        - ma_q35:
-            ma_type = "q35"
-        - ma_i440fx:
-            only by_seabios
-            ma_type = "i440fx"
```
Comment (kylazhang) [inline libvirt/tests/cfg/bios/boot_integration.cfg]: Need to add only i440fx in by_seabios

File: libvirt/tests/cfg/bios/virsh_boot_tseg.cfg [hunk 2]
```diff
@@ -47,14 +47,9 @@
...
+            only q35
             boot_type = "ovmf"
             loader = "/usr/share/OVMF/OVMF_CODE.secboot.fd"
             loader_type = "pflash"
         - by_seabios:
             boot_type = "seabios"
-    variants:
-        - ma_q35:
-            only q35
-        - ma_i440fx:
```
Comment (kylazhang) [inline libvirt/tests/cfg/bios/virsh_boot_tseg.cfg]: In the file line 29 need to remove and add only i440fx

File: libvirt/tests/cfg/bios/boot_integration.cfg (hunk 1 above)
Comment (kylazhang) [inline libvirt/tests/cfg/bios/boot_integration.cfg]: Here previous setting is only by_seabios but not sure if it needed to test with q35, need to check with feature owner
If not needed, need to add 'only i440fx' in by_seabios part

File: libvirt/tests/cfg/bios/virsh_boot_tseg.cfg (hunk 2 above)
Comment (brucepatricklee) [reply libvirt/tests/cfg/bios/virsh_boot_tseg.cfg]: good point, updated

File: libvirt/tests/cfg/bios/boot_integration.cfg (hunk 1 above)
Comment (brucepatricklee) [reply libvirt/tests/cfg/bios/boot_integration.cfg]: The previous setting means i440fx only support seabios and confirmed with feature owner, the seabios feature support both i440fx and q35 machine

**Pull request merged**